aptos-move/vm-genesis/src/lib.rs

```diff
@@ -562,7 +559,7 @@ pub fn test_genesis_module_publishing() {
     }
     let data_cache = StateViewCache::new(&state_view).into_move_resolver();
 
-    let move_vm = MoveVmExt::new().unwrap();
+    let move_vm = MoveVmExt::new(NativeGasParameters::zeros()).unwrap();
```

Comment:
Does this make sense to charge no gas in tests though? Are you planning on adding e2e tests for gas costs in a followup PR?

Reply:
The vm-genesis crate is strictly a tool to help us generate a genesis blob. It does use the Move VM, but gas metering is 100% meaningless in this context since you want all operations to succeed.

Coming to the test itself. I think it's actually questionable whether this test makes sense or not.